README.md

```diff
@@ -47,6 +47,13 @@ saucectl configure -u <MyUser> -a <MyAccessKey>
 
 The credentials are store in `$HOME/.sauce/credentials.yml`
 
+## The `signup` Command
```

Comment:
Might it make sense to move this command and the `configure` command above the `run` command?

The Flags section below has much more to do with the `run` command than any others.